lib/index.js

```diff
@@ -15,11 +15,15 @@ const defaults = {
     server: {
       host: 'http://localhost:5000'
     },
-    reporting: {
-      method: 'POST',
-      url: 'http://localhost:5000/report',
-      options: {}
-    }
+    reporters: [
+      {
+        // TODO: The reporters can be separate npm modules, required by name
```

Comment:
Once we get to around a 1.0.0, I'd like to publish the different reporters and plugins as separate npm modules. Then, in your config file, it would be the module name to require in and configure.